packages/server/test/unit/plugins/child/validate_event_spec.js

```diff
@@ -20,21 +20,7 @@ describe('lib/plugins/child/validate_event', () => {
     const { isValid, error } = validateEvent()
```

Comment:
Should we add more tests to cover the new functionality added to `validateEvent`, like setting `error.name` to `ValidationError` and the use of `Error.captureStackTrace`?

Reply:
probably